source/themes/ThemeContext.js

```diff
@@ -8,8 +9,23 @@ import ROOT_THEME_API from './API';
 // same goes for "this.props.context.ROOT_THEME_API"
 // if the user passes ThemeProvider a theme and/or ROOT_THEME_API,
 // these default values are overwritten
-// $FlowFixMe
-export const ThemeContext = React.createContext({
+
+type Theme = {
+  theme: Object,
+  ROOT_THEME_API: Object
+};
+
+// check for createContext method on React
+let createContext;
+if (React.createContext) {
```

Comment:
Here we manually check if createContext is available on React before using the polyfill.